src/components/JUPTable/JUPTable.tsx

```diff
@@ -103,10 +103,11 @@ interface IJUPTableProps {
   path?: string;
   DisplayedComponents?: Array<React.ReactElement>;
   defaultSortOrder?: Order;
+  isPaginated?: boolean;
```

Comment:
just now thinking about conventions in naming schemes for something like this. kinda like how we have a convention for how `useState` has the `setFoo` convention. maybe booleans used for enabling/disabling have the convention where they start with `enable`, aka for this example `enablePagination`

super small thing, more general advice than anything. :)

Reply:
i've been trying to stick to "is<something>" to indicate it's a boolean, so I was already trying to standardize somehow (I haven't always followed my own thoughts here probably)

enable is probably better